src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/WebWorkerTest.Http.cs

```diff
@@ -82,8 +82,22 @@ public async Task HttpClient_CancelInDifferentThread(Executor executor1, Executo
                 {
                     CancellationTokenSource cts = new CancellationTokenSource();
                     var promise = response.Content.ReadAsStringAsync(cts.Token);
-                    cts.Cancel();
-                    await promise;
+                    try
```

Comment:
please move as well `ReadAsStringAsync` inside the try-catch

Reply:
Moved.